addons/mail/tests/common.py

```diff
@@ -1951,6 +1951,16 @@ def _activate_multi_lang(cls, lang_code='es_ES', layout_arch_db=None, test_recor
             }
         })
 
+    def _filter_channels_fields(self, /, *channels_data):
```

Comment:
Usage of `_filter_channels_fields` and `livechat_with_ai_agent` key must be added in **ALL** tests where `"channel_type": "livechat"` is present in payload in **ALL** modules, otherwise these tests will fail when `ai_livechat` is installed.

You won't necessarily see it with standard runbot, they will either fail in nightly or when running them locally with that setup.